lib/vagrant-ca-certificates/cap/redhat/update_certificate_bundle.rb

```diff
@@ -9,7 +9,13 @@ module UpdateCertificateBundle
           def self.update_certificate_bundle(m)
             m.communicate.tap do |sh|
               if Redhat.legacy_certificate_bundle?(sh)
-                sh.sudo('find /etc/pki/tls/private -type f -exec cat {} \; | cat /etc/pki/tls/certs/ca-bundle.crt - > /etc/pki/tls/ca.private.crt')
+                sh.sudo(<<-SCRIPT)
+BUNDLE=/etc/pki/tls/certs/ca-bundle.crt; 
+PRIVATE=/etc/pki/tls/ca.private.crt; 
+if ! [ "$(readlink $BUNDLE)" == "$PRIVATE" ]; then 
```

Comment:
It'd probably be more robust to keep a proper copy of the ca bundle and reconstruct it on each run. As written adding a new certificate won't work as expected (it'll see it's already linked and do nothing).

For this case, I don't believe running the links repeatedly has any effect.